python/res/enkf/config/enkf_config_node.py

```diff
@@ -177,3 +403,82 @@ def getModelConfig(self):
 
     def getKey(self):
         return self._get_key()
+
+    def __ne__(self, other):
+        return not self == other
+
+
+    def __eq__(self, other):
```

Comment:
This function is quite long and a bit hard to read. Perhaps it could be broken down into separate helper functions?

Reply:
Maybe